src/_P013_HCSR04.ino

```diff
@@ -42,12 +44,12 @@ boolean Plugin_013(byte function, struct EventStruct *event, String& string)
       {
         Device[++deviceCount].Number = PLUGIN_ID_013;
         Device[deviceCount].Type = DEVICE_TYPE_DUAL;
-        Device[deviceCount].VType = Sensor_VType::SENSOR_TYPE_SINGLE;
+        Device[deviceCount].VType = SENSOR_TYPE_DUAL;
```

Comment:
Should be using the `Sensor_VType::` prefix.

Reply:
Yep, sure, otherwise it won't compile anyway… I changed it but did not yet commit…